code/modules/clothing/under/jobs/medsci.dm

```diff
@@ -92,6 +141,15 @@
 	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 10, rad = 0)
 	flags_size = ONESIZEFITSALL
 
+/obj/item/clothing/under/rank/virologist/ncu
+	desc = "A Nanomesh Combat Uniform. This one is fitted for Medical personnel."
+	name = "virologist's nanomesh combat uniform"
+	icon_state = "ncuvirologywhite"
+	item_state = "ncuvirologywhite"
+	item_color = "ncuvirologywhite"
+	permeability_coefficient = 0.50
+	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 10, rad = 0)
+	flags_size = ONESIZEFITSALL
 /obj/item/clothing/under/rank/virologist/skirt

```

Comment:
I have yet to look it over for fixing but I expected this much stuff when coding.